README.md

```diff
@@ -1 +1,33 @@
-# nanoexp-digitalbank
\ No newline at end of file
+# nanoexp-digitalbank
+
+
+## **Brief introduccion **
+This readme file mentions the requirements and establishes a guide for running those tests.
+Getting Started:
+
+
+## Prerequisites
+1. Install Java SDK
+2. Install node.js
+3. Node Package Manager (npm)
+
+4. Install Visual Studio Code IDE
+⋅⋅* wdio.config
+⋅⋅* npm install @wdio/cli
+⋅⋅* npm install chai
+⋅⋅* npm install chai-webdriverio
```

Comment:
This will not be needed if you have them on your `package.json`, as users only need to do a general `npm install`, when their working directory is your project directory, and they'll be installing the same dependencies you are using.